### PR TITLE
Changes to Stocks Console UI, cargo fixes

### PR DESCRIPTION
**Pull request by Core0verload**
This PR changes Stocks Console compact mode UI to make it suck a tiny bit less.
stock
It also includes fixes for a pair of cargo/exports bugs and a money bag refactor (it's a real storage bag now).

It changes various instances of "cargo points" to "credits" for the sake of consistency.

Offtopic: Who added "XUY" to possible companies list?

### CODE REVIEW

File: code/modules/stock_market/computer.dm [hunk 1]
```diff
@@ -190,30 +230,25 @@ a.updated {
...
 	b = balance()
 	if (!isnum(b))
 		user << "<span class='danger'>No active account on the console!</span>"
 		return
-	if (amt > S.available_shares)
-		user << "<span class='danger'>That many shares are not available!</span>"
-		return
-	var/total = amt * S.current_value
-	if (total > b)
-		user << "<span class='danger'>Insufficient points.</span>"
```
Comment (coiax) [inline code/modules/stock_market/computer.dm]: Why is this check no longer in? Wouldn't you be able make transactions without the required credits?

**Comment by GunHog** [comment]
It looks so much like TGUI that it might as well be transferred to it, given that stocks need refreshing often.

**Comment by Iamgoofball** [comment]
hnngh

:+1:

**Comment by WJohn** [comment]
Tgui with stocks graphing prediction when?

**Comment by Core0verload** [comment]
When TGUI will stop glitching out and breaking literally for no reason, will be optimized to not take ~5 seconds to load even the simplest UI and drop all that "install a nodejs to make new uis" bullshit.

File: code/modules/stock_market/computer.dm (hunk 1 above)
Comment (Core0verload) [inline code/modules/stock_market/computer.dm]: They are clamped by amount of credits you have now. You can type 99999 in sell or buy to sell or buy as many stocks as you can. A little QoL change.

**Comment by Core0verload** [comment]
Merge anyone?

**Pull request merged**